### ISSUE TITLE
Change format of charts

### ISSUE THREAD

**Issue by sebgoa** [helm/charts#7]
change format of merged charts from helm/charts to new unified format

depends on #6

**Comment by jackgr** [comment]
@runseb a quick sampling of the charts suggests that most if not all charts are already in the appropriate format. Can this issue now be closed?

**Comment by sebgoa** [comment]
@jackgr we need to close the loop on the expander name field.
Currently none of the charts can actually be used because there is no expander.

And if we added the expander-service to them, still the deployments would fail.

I think we should leave this open until we can actually run them.

**Comment by jackgr** [comment]
@runseb I've added #11 to address the Expander issue for charts w/DM templates. 

The rest should work fine with no Expander field, since the default is to treat the contents as Kubernetes API objects. Will confirm when I get the time. That said, it would be nice to have an automated test framework for charts that deploys them to a playground. 

Also, not sure I follow your point that they would still fail if we added expander-service.
